engine/src/main/scala/cromwell/util/TryUtil.scala

```diff
@@ -4,13 +4,14 @@ import java.io.{PrintWriter, StringWriter}
 
 import cromwell.engine.CromwellFatalException
 import cromwell.logging.WorkflowLogger
+import org.apache.commons.lang3.exception.ExceptionUtils
 
 import scala.language.postfixOps
 import scala.util.{Failure, Success, Try}
 
 case class AggregatedException(exceptions: Seq[Throwable], prefixError: String = "") extends Exception {
   override def getMessage: String = {
-    prefixError + exceptions.map(_.getMessage).mkString("\n")
+    prefixError + exceptions.map(e => s"${e.getMessage}\n${ExceptionUtils.getStackTrace(e)}").mkString("\n")
```

Comment:
I know.. The problem with this AggregatedException is that is concatenates the messages but you lose all the stacktraces. I know it's ugly but it makes it hard sometimes to know what happened when postProcessing fails for example. We just get these "500 Internal Server Error" messages with no real way to know which part of the processing failed.
If you have a better idea I'm all ears ! :)